docs/administration-guide/databases/oracle.md

```diff
@@ -1,67 +1,52 @@
-## Working with Oracle in Metabase
+# Working with Oracle in Metabase
 
-Starting in v0.20.0, Metabase provides a driver for connecting to Oracle databases. Under the hood, Metabase uses Oracle's JDBC driver; due to licensing restrictions, we can't
-include it as part of Metabase. Luckily, downloading it yourself and making it available to Metabase is straightforward and only takes a few minutes.
+Under the hood, Metabase uses Oracle's JDBC driver, but due to licensing restrictions, we can't include Oracle's driver as part of Metabase, so you'll need to download the driver and make it available to Metabase.
 
-### Downloading the Oracle JDBC Driver JAR
+## Downloading the Oracle JDBC Driver JAR
 
-You can download the JDBC driver from [Oracle's JDBC driver downloads page](https://www.oracle.com/technetwork/database/application-development/jdbc/downloads/index.html).
-Head to this page, accept the license agreement, and download `ojdbc8.jar`:
+You can download a JDBC driver from [Oracle's JDBC driver downloads page](https://www.oracle.com/technetwork/database/application-development/jdbc/downloads/index.html).
 
-![Oracle JDBC Download](../images/oracle_jdbc_download.png)
+The minimum driver version should be 19c, regardless of which Java version or Oracle version you have.
 
-Before downloading this JAR you may need to sign up for a free account with Oracle. We have had success with the latest version at the time of this writing, 19.3 (even with older Oracle 12c databases), but any version _should_ work.
+We recommend using the `ojdbc11.jar` JAR.
```

Comment:
Lets continue to recommend what we're shipping in EE builds. Once we change that, then we change the docs.
```suggestion
We recommend using the `ojdbc8.jar` JAR.
```